src/linux/osd.c

```diff
@@ -71,9 +71,9 @@ ssize_t ofi_get_hugepage_size(void)
 	return val * 1024;
 }
 
-#ifdef HAVE_ETHTOOL
+#if HAVE_ETHTOOL
```

Comment:
This should still be ifdef, unless you update configure.ac accordingly.

Reply:
@shefty thanks, I missed it